src/containers/boards/Article/ArticleList.tsx

```diff
@@ -9,14 +10,19 @@ import Button from '@/src/components/Button/Button'
 import styles from './Article.module.scss'
 
 export default function ArticleList() {
-  const [orderBy, setOrderBy] = useState('recent')
+  type OrderOption = 'recent' | 'like'
+  const [orderBy, setOrderBy] = useState<OrderOption>('recent')
   const [searchTitle, setSearchTitle] = useState('')
-  const fetechArticles = useFetechData<ArticleListResponse>(
-    `articles?page=1`,
-    10,
-    orderBy,
-    searchTitle
-  )
+
+  const url = useMemo(() => {
+    let baseUrl = `articles?page=1&pageSize=10&orderBy=${orderBy}`
+    if (searchTitle) {
+      baseUrl += `&keyword=${searchTitle}`
+    }
+    return baseUrl
+  }, [orderBy, searchTitle])
+
+  const fetechArticles = useFetechData<ArticleListResponse>(url)
   const { data: ArticleList, isLoading } = fetechArticles
```

Comment:
```suggestion
  const { data: articleList, isLoading } = useFetchData<ArticleListResponse>(url)
```
하나로 합칠 수 있을 것 같아요!
그리고 변수이름은 camelCase로 해주는 것이 일반적입니다~ 오타도 하나 있네요!